traffic_ops/traffic_ops_golang/ping/keys.go

```diff
@@ -20,25 +20,29 @@ package ping
  */
 
 import (
-	"errors"
+	"github.com/apache/trafficcontrol/lib/go-tc"
 	"net/http"
 
 	"github.com/apache/trafficcontrol/traffic_ops/traffic_ops_golang/api"
 	"github.com/apache/trafficcontrol/traffic_ops/traffic_ops_golang/riaksvc"
 )
 
 func Keys(w http.ResponseWriter, r *http.Request) {
+	alerts := tc.CreateAlerts(tc.WarnLevel, "This endpoint is deprecated")
 	inf, userErr, sysErr, errCode := api.NewInfo(r, nil, nil)
 	if userErr != nil || sysErr != nil {
-		api.HandleErr(w, r, inf.Tx.Tx, errCode, userErr, sysErr)
+		userErr = api.LogErr(r, errCode, userErr, sysErr)
+		alerts.AddAlerts(tc.CreateErrorAlerts(userErr))
+		api.WriteAlerts(w, r, errCode, alerts)
```

Comment:
so i added a HandleDeprecatedErr function in PR 4452 that will hopefully be merged in soon but id use that generic handler instead. i can see about getting it reviewed / merged otherwise you can just steal that code